source/slang/hlsl.meta.slang

```diff
@@ -9571,8 +9571,10 @@ void CallShader(uint shaderIndex, inout Payload payload)
 
 // 10.3.2
 
-// This intrinsic returns `maybeStruct` if a struct. If `maybeStruct` is a non struct it returns a `struct{T maybeStruct};` object
-// the intrinsic will copy `struct{T maybeStruct};` into `maybeStruct` if the parameter can exhibit side effects (`inout`, `out`, `ref`, etc.)
+// Some functions only accept a `struct type` parameter. The 
```

Comment:
This is a nitpick and feel free to ignore for this case.
The back-tick is for referring a code snippet; in Doxygen notation.
You can use double quote here for "struct type".